query/readme.md

```diff
@@ -1,5 +1,8 @@
 # Query API
 
+## DEPRECATION NOTICE - Dec 2016
+We're deprecating this Query API in favor of Search API which is a superset of this API. Please refrain from using it.
```

Comment:
We're deprecating this existing Query API and superseding it with a new Search API. Please refrain from using it.